### PR TITLE
add video play and pause icons

### PR DESCRIPTION
**Pull request by eatdesignlove**
## I have read the [CONTRIBUTING.md](https://github.com/pqoqubbw/icons/blob/main/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Add icons

## What is the new behavior?

Add play and pause icons

## Demo

https://github.com/user-attachments/assets/3912b181-12b5-43e7-8455-87446c84ac94

## Additional context

I’m also preparing other icons for the video player. I’m participating in this project with a light heart, but I think there should be guidelines for icon style and interaction. What do you think?

스크린샷 2024-11-09 오전 9 51 58

### CODE REVIEW

File: icons/play.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const pathVariants: Variants = {
+  normal: {
+    opacity: 1,
+    x: 0,
+    rotate: 0,
+  },
+  animate: {
+    x: [0, -1, 2, 0],
+    rotate: [0, -10, 0, 0],
+    opacity: 0.5,
```
Comment (pqoqubbw) [inline icons/play.tsx:15]: why did you decide to change the opacity? Ii don't think we should change it, it's all up to the user

**Comment by pqoqubbw** [comment]
> I’m also preparing other icons for the video player. I’m participating in this project with a light heart, but I think there should be guidelines for icon style and interaction. What do you think?

how do you imagine this? i don't quite understand. you mean, like, doc "how to change the animation of [icon]"?

File: icons/play.tsx (hunk 1 above)
Comment (eatdesignlove) [reply icons/play.tsx:15]: @pqoqubbw Oh, I’m sorry. I didn’t mean to do that either, and it was something I briefly applied during testing, but it ended up being included in the PR. I’ll make the necessary updates!

**Comment by eatdesignlove** [comment]
> > I’m also preparing other icons for the video player. I’m participating in this project with a light heart, but I think there should be guidelines for icon style and interaction. What do you think?
> 
> how do you imagine this? i don't quite understand. you mean, like, doc "how to change the animation of [icon]"?

I’ve been adding icons by first designing them in Figma, extracting the SVG, and then applying Framer Motion in the code editor. During this process, I had two main concerns:

	1.	What are the key things to consider in order to create icons that maintain the same tone and manner as the existing ones? (e.g., frame size, minimum padding etc in figma)
	2.	Is there a direction to follow when applying animations to the icons?

These were my two main concerns.

As many people, like myself, are interested in this project, the direction will likely become clearer with further contributions from various people. However, I thought there might be others with similar concerns, which is why I wanted to ask if a guide for these two concerns might be needed🙂

**Comment by pqoqubbw** [comment]
> > > I’m also preparing other icons for the video player. I’m participating in this project with a light heart, but I think there should be guidelines for icon style and interaction. What do you think?
> > 
> > 
> > how do you imagine this? i don't quite understand. you mean, like, doc "how to change the animation of [icon]"?
> 
> I’ve been adding icons by first designing them in Figma, extracting the SVG, and then applying Framer Motion in the code editor. During this process, I had two main concerns:
> 
> ```
> 1.	What are the key things to consider in order to create icons that maintain the same tone and manner as the existing ones? (e.g., frame size, minimum padding etc in figma)
> 2.	Is there a direction to follow when applying animations to the icons?
> ```
> 
> These were my two main concerns.
> 
> As many people, like myself, are interested in this project, the direction will likely become clearer with further contributions from various people. However, I thought there might be others with similar concerns, which is why I wanted to ask if a guide for these two concerns might be needed🙂

regarding icon creation and consistency:
currently, we use lucide icons in our project rather than creating custom ones. lucide is a comprehensive icon library that covers most of our needs and has gained significant popularity in the developer community. we just use their svg code and focus on adding animations when needed. i haven't specifically focused on custom icon design guidelines cause i don't really know how to design proper icons

regarding icon animations:
the approach to animations is quite flexible. they are typically created either by me or project contributors without strict rules. the key principles are:
- animations should feel appropriate and natural in the context they're used
- we provide animation code as templates that users can copy and modify
- since different projects may require different animation styles, we encourage users to adapt the animation code to match their specific project needs

this flexible approach allows developers to maintain consistency with their project's design language while leveraging our animation examples as starting points

**Comment by eatdesignlove** [comment]
@pqoqubbw Thank you so much for the detailed response! 

As you mentioned, if this project is moving forward with adding animations to Lucide icons, then my pull request may not be helpful. The icons in this PR are based on custom-made SVGs. If you agree with this, I will go ahead and close this PR.

I will also contribute again in the future with animated icons based on Lucide.

**Comment by pqoqubbw** [comment]
> @pqoqubbw Thank you so much for the detailed response!
> 
> As you mentioned, if this project is moving forward with adding animations to Lucide icons, then my pull request may not be helpful. The icons in this PR are based on custom-made SVGs. If you agree with this, I will go ahead and close this PR.
> 
> I will also contribute again in the future with animated icons based on Lucide.

oh, didn't notice that this is a custom svg, my bad. i guess there's no need to close the PR - you can just adapt the lucide svg, or i can do this if you want. but thanks a lot for the contribution, love the animations ❤️

on my part, i'll think about how to handle custom icons and add this to the templates so that anyone can add their own icons if they want

**Comment by eatdesignlove** [comment]
I’ve updated it to use the Lucide SVG!
I’m really happy with my first contribution.

**Comment by eatdesignlove** [comment]
> > @pqoqubbw Thank you so much for the detailed response!
> > As you mentioned, if this project is moving forward with adding animations to Lucide icons, then my pull request may not be helpful. The icons in this PR are based on custom-made SVGs. If you agree with this, I will go ahead and close this PR.
> > I will also contribute again in the future with animated icons based on Lucide.
> 
> oh, didn't notice that this is a custom svg, my bad. i guess there's no need to close the PR - you can just adapt the lucide svg, or i can do this if you want. but thanks a lot for the contribution, love the animations ❤️
> 
> on my part, i'll think about how to handle custom icons and add this to the templates so that anyone can add their own icons if they want

I’ve updated it to use the Lucide SVG!
I’m really happy with my first contribution😆

**Pull request merged**